lib/shifts.rb

```diff
@@ -0,0 +1,66 @@
+require 'date'
+
+class Shifts
+
+    def initialize
+      @shifts = shifts
+      @keys = keys
```

Comment:
Might have a problem when key is preloaded. Something to face when that problem arises.